USAGE.md

```diff
@@ -168,6 +169,10 @@ Note: Dynamic Updates marked with an asterisk (*) are enabled by the `AuthAwareC
 * `manta.skip_directory_depth` (**MANTA_SKIP_DIRECTORY_DEPTH**)
     Integer indicating the number of **non-system** directory levels to attempt to skip for recursive `putDirectory`
     operation (i.e. `/$MANTA_USER` and `/$MANTA_USER/stor` would not be counted). A detailed explanation and example are provided [later in this document](/USAGE.md#skipping-directories)
+* `manta.prune_empty_parent_depth` (**MANTA_PRUNE_EMPTY_PARENT_DEPTH**)
+    Integer indicating the maximum number of empty parent directories to prune. If the client is given the value of -1 then the client will try and delete
+    all empty parent directories.
+    USAGE.md#Pruning empty parent directories)
```

Comment:
`USAGE.md#Pruning empty parent directories)`

looks like a typo, I assume this was supposed to be something like

`see [the section on parent directory pruning later in this document](/USAGE.md#Pruning empty parent directories)`